### PR TITLE
Add Db2

### PR DESCRIPTION
**Pull request by angoca**
https://github.com/angoca/awesome-db2/#README.md

This is the awesome stuff about Db2, an IBM database. This page has already different types of resources, and it is based on the already existing db2-index (https://github.com/angoca/db2-index/wiki) that has been for 3 years (I maintain that). The awesome list is curated with the main resources from the db2-index. The list was created by a DBA DB2 with a lot of experience in this field, and recognized as an IBM Champion 2015-2019 (me).

I know that the list has less that one day since the first commit, but it is based on an old list. Also, I created the pull request to start improving the list with your comments.

I read the list of requirements several times, and I did what it says.

Finally, I also sorted the entries related to databases in the README file, in order to ease the access.

Best regards,

Andres Gomez

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by victorshinya** [comment]
Thanks for contributing @angoca 

Take a look for quick things:
- Your repository needs to have at least 30 days old and the first contribution was made less than 1 day
- I would suggest you change the LICENSE content for the official one. Take a look at this repository as an example (https://github.com/victorshinya/awesome-ibmcloud/blob/master/LICENSE)

**Comment by angoca** [comment]
> Thanks for contributing @angoca
> 
> Take a look for quick things:
> 
> * Your repository needs to have at least 30 days old and the first contribution was made less than 1 day
I know it is quite new, but as I stated in the PR, my awesome list is based on a curated list that has more than 4 years (db2-index). Anyway, I created the PR to have feedback as your comments.
> * I would suggest you change the LICENSE content for the official one. Take a look at this repository as an example (https://github.com/victorshinya/awesome-ibmcloud/blob/master/LICENSE)
About the license, in this link https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license, it recommends CC licenses. CC-0 is too open for me, and for this reason I prefer CC-SA, the attribution and the share-alike derivatives.

File: readme.md [hunk 1]
```diff
@@ -392,17 +392,18 @@
...
 - [MongoDB](https://github.com/ramnes/awesome-mongodb#readme) - NoSQL database.
+- [MySQL](https://github.com/shlomi-noach/awesome-mysql/blob/gh-pages/index.md)
+- [Neo4j](https://github.com/neueda/awesome-neo4j#readme)
+- [PostgreSQL](https://github.com/dhamaniasad/awesome-postgres#readme) - Object-relational database.
 - [RethinkDB](https://github.com/d3viant0ne/awesome-rethinkdb#readme)
+- [SQLAlchemy](https://github.com/dahlia/awesome-sqlalchemy#readme)
 - [TinkerPop](https://github.com/mohataher/awesome-tinkerpop#readme) - Graph computing framework.
-- [PostgreSQL](https://github.com/dhamaniasad/awesome-postgres#readme) - Object-relational database.
-- [CouchDB](https://github.com/quangv/awesome-couchdb#readme) - Document-oriented NoSQL database.
-- [HBase](https://github.com/rayokota/awesome-hbase#readme) - Distributed, scalable, big data store.
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

File: readme.md (hunk 1 above)
Comment (angoca) [reply readme.md]: I remove the sorting of this entries. I left them as they were before.

**Comment by angoca** [comment]
I remove the unrelated changes. Now, the pull request only includes the new entry related to db2.

**Comment by angoca** [comment]
Any news?

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/angoca/awesome-db2
✖ Linting

  README.md:13:3
  ✖    1:1   License section must be at heading depth 2                    remark-lint:awesome/license
  ✖   13:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   14:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   15:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   20:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   20:94  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   21:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   21:94  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   22:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   22:94  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   23:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   23:92  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   24:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   27:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   30:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   31:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   32:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   33:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   34:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   35:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   36:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   37:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   38:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   39:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   40:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   43:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   44:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   47:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   48:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   49:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   50:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   51:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   52:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   53:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   54:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   55:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   58:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   58:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   58:62  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖   59:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   59:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   60:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   60:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   61:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   61:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   62:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   62:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   63:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   63:3   Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ✖   66:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   66:3   Link to https://idug.org/ is dead                             remark-lint:no-dead-urls
  ✖   71:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   74:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   75:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   76:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   77:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   80:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   81:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   84:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   85:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   86:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   87:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   91:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   92:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   93:1   Marker style should be -                                      remark-lint:unordered-list-marker-style
  ✖   95:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings
  ✖  101:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings

  67 errors

```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
The heading of your Awesome list should not be in slug format. `Awesome Unicorn`, not `awesome-unicorn`.

**Comment by sindresorhus** [comment]
> * If you use a license badge, it should be SVG, not PNG.

**Comment by sindresorhus** [comment]
> * Be added at the bottom of the appropriate category.

File: readme.md [hunk 2]
```diff
@@ -395,6 +395,7 @@
 ## Databases
 
 - [Database](https://github.com/numetriclabz/awesome-db#readme)
+- [Db2](https://github.com/angoca/awesome-db2#readme)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

**Comment by sindresorhus** [comment]
> * **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.

**Comment by angoca** [comment]
Updated with your new comments.

angoca@ip-10-0-0-165:~/awesome-db2$ awesome-lint 
✔ Linting

**Comment by sindresorhus** [comment]
You still need to do https://github.com/sindresorhus/awesome/pull/1490#issuecomment-467499664.

**Comment by AdrienTorris** [comment]
Maybe you can add another topics if they are relevants, like 'db2-warehouse'.

**Comment by angoca** [comment]
As part of the pull request template, contributors should review at least two other pull request in order "to help make the Awesome project self-sustaining"; however, you @sindresorhus is the only that performs the pull requests. This is contradictory, and this does not promote a community trying to make the main awesome repository something great. People just want to have their own repositories listed here and then they just forget this repository.
Also, old pull request included in the main repository that links to external repositories in most of the cases are very different to the continuously changing rules for new repositories. In other words, there is not a maintenance for old repositories that do not comply with the new rules, like removing them from the awesome list because they do not comply with the new rules.
